src/test/java/com/epam/tc/hw1/TestForDivideByZeroLongFunction.java

```diff
@@ -0,0 +1,35 @@
+package com.epam.tc.hw1;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import com.epam.tat.module4.Calculator;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+public class TestForDivideByZeroLongFunction {
+
```

Comment:
Correct code in that class.
Don't initialise exception as a class field.
Take a look on testNG documentation or use assertThatThrownBy from assertJ